src/pages/config/config-list.js

```diff
@@ -51,8 +44,30 @@ export function ConfigListPage() {
     }
   };
 
+  const extra = (
+    <div className="flex space-x-2 mr-4">
+      <TypeDropdown />
+      <TagsDropdown />
+      <TextInputClearable
+        onChange={debounce((e) => {
```

Comment:
This will creates new function each time this block is executed. Could you please refactor this using useCallback hook.

Reply:
https://kentcdodds.com/blog/usememo-and-usecallback Let's optimize when we see a need for it? Right now, I don't think the interactions has performance bottleneck.